uiplib/GUI.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, settings):
         self.settings = settings
         # base window
         self.root = Tk()
-        self.root.resizable(width=False, height=False)
+        self.root.resizable(width=True, height=True)
```

Comment:
@lohani2280 yes everything should keep its relative positions, idk if we even want this, do we @nkprince007 fixed things let us keep more control and let us have one less thing to worry about in the future